test5.js

```diff
@@ -0,0 +1,12 @@
+var foo = '' ;
+
+if(foo  === "bar") {
+  console.log( "hey" )
+}
+
+
+var foo77 = '' ;
+
+if(foo77  === 'baaaaaaaaaar') {
```

Comment:
Codacy has a fix for the issue: [Multiple spaces found before '==='.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26923)
```suggestion
if(foo77 === 'baaaaaaaaaar') {
```